### PR TITLE
Feat/swarinfo 413 push interloc to buzz

### PR DESCRIPTION
**Pull request by cquesnel**
Make interloc data go up the stack into buzz

### CODE REVIEW

File: src/bsp/src/stm32/src/interloc/include/interloc/InterlocStateHandler.h [hunk 1]
```diff
@@ -51,9 +54,7 @@ class InterlocStateHandler {
     CircularQueue<StateTransition> m_stateTracer;
 
     uint8_t m_sequenceID = 0;
-
-    // TODO : set manually /AprilTag
```
Comment (t-constantin) [inline src/bsp/src/stm32/src/interloc/include/interloc/InterlocStateHandler.h:55]: How is it handled right now? (Not an issue, just curious)

File: src/bsp/src/stm32/src/interloc/src/states/src/SetDistanceState.cpp [hunk 2]
```diff
@@ -12,8 +12,11 @@ void SetDistanceState::process(InterlocStateHandler& context) {
     if (distance) {
         m_logger.log(LogLevel::Warn, "distance from : %d = %3.3f", context.getCurrentFrameId(),
                      distance.value());
+
+        InterlocBSPContainer::getInterlocManager().updateDistance(
+            context.getBoardIdFromSlotId(context.getCurrentFrameId()), distance.value());
+
         context.setState(InterlocStates::IDLE, InterlocEvent::GOOD_DISTANCE);
-        //        TODO notify Buzz that a new distance value is available
```
Comment (t-constantin) [inline src/bsp/src/stm32/src/interloc/src/states/src/SetDistanceState.cpp:16]: Before removing the to do, is it already done?

**Review by t-constantin — APPROVED** [review]
LGTM! Clean code as we like it
Just few questions, but nothing blocking

**Review by g-lesssard — APPROVED** [review]
LGTM :+1:

**Review by xgroleau — APPROVED** [review]
LGTM

File: src/bsp/src/stm32/src/interloc/include/interloc/InterlocStateHandler.h (hunk 1 above)
Comment (cquesnel) [reply src/bsp/src/stm32/src/interloc/include/interloc/InterlocStateHandler.h:55]: We use the board id (the same that is returned when you greet with the HM). In the next days, we will embed this ID directly in the Nucleo's memory so they will each have a unique ID from 1 to 5

**Pull request merged**